frontend/lib/src/util/utils.ts

```diff
@@ -322,7 +322,7 @@ export function setCookie(
 
 /** Return an Element's widget ID if it's a widget, and undefined otherwise. */
 export function getElementWidgetID(element: Element): string | undefined {
-  return _.get(element as any, [requireNonNull(element.type), "id"])
+  return get(element as any, [requireNonNull(element.type), "id"])
```

Comment:
We may be able to update this to:

```ts
return element?.[requireNonNull(element.type)]?.id
```

This would allow us to get rid of the `lodash/get` import in this file.